requirements/production.txt

```diff
@@ -1,2 +1,3 @@
 -r base.txt
+
 MySQL-python==1.2.5
```

Comment:
Can we *please* talk about "which production database?"? o.O

Reply:
I think, it's not specific to the update to Django 4. Let's address it separately.